dartagnan/src/test/java/com/dat3m/dartagnan/compilation/C11ToPPCTest.java

```diff
@@ -49,9 +50,12 @@ protected Provider<Arch> getTargetProvider() {
 	protected List<String> getCompilationBreakers() {
 		return Stream.of(
 				"manual/IRIW-sc-sc-acq-sc-acq-sc",
-				"manual/RWC-sc-acq-sc-sc-sc").map(p -> ResourceHelper.LITMUS_RESOURCE_PATH + "litmus/C11/" + p + ".litmus").collect(Collectors.toList());
+				"manual/RWC-sc-acq-sc-sc-sc")
+				.map(p -> ResourceHelper.LITMUS_RESOURCE_PATH + "litmus/C11/" + p + ".litmus")
+				.map(p -> Path.of(p).toString())
```

Comment:
This is a quick fix for windows, but I think it needs a proper refactoring. It is not a good practice to hardcode test case names this way,